service/k8sclient/k8sclient_test.go

```diff
@@ -82,7 +83,7 @@ func TestK8sClient(t *testing.T) {
 
 		l.Info("No XtraDB Clusters running")
 
-		err = client.CreateXtraDBCluster(ctx, &XtraDBParams{
+		err := client.CreateXtraDBCluster(ctx, &XtraDBParams{
```

Comment:
We need this to avoid data race on err variable. We run psmdb and pxc tests in parallel to speed up tests.